tests/ui/pages/data_entity/overview.page.ts

```diff
@@ -0,0 +1,54 @@
+import Button from '../../elements/button';
+import InputField from '../../elements/input-field';
+import DataEntityPage from './data_entity.page';
+
+const SELECTORS = {
+  addOwnerButton: `text=Add Owner`,
+  inputOwnerName: `[placeholder="Search name"]`,
+  inputOwnerTitle: `[placeholder="Search title"]`,
+  autocomplete: text => `[role="presentation"]:has-text('${text}')`,
+  saveNewOwnerButton: `[type="submit"]`,
+  addTagButton: `.MuiButton-root.MuiButton-text.MuiButton-textPrimary.MuiButton-sizeSmall:has-text('Add Tags')`,
```

Comment:
Pls get rid of unneeded classes in SELECTORS.addTagButton

Reply:
all of these classes are needed because when I remove the last classes there are two buttons with the same name